.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,99 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
I am not able to verify it quickly, but all of our projects use pretty much the same standardized codestyle configuration. Again, please see `kanban` for reference.

Reply:
I have copied it from kanban, not sure what can be wrong.